main.go

```diff
@@ -27,9 +28,11 @@ func main() {
 	}
 
 	// main message loop
-	var msg win.MSG
-	for win.GetMessage(&msg, 0, 0, 0) > 0 {
-		win.TranslateMessage(&msg)
-		win.DispatchMessage(&msg)
+	msg := (*win.MSG)(unsafe.Pointer(win.GlobalAlloc(0, unsafe.Sizeof(win.MSG{}))))
+	defer win.GlobalFree(win.HGLOBAL(unsafe.Pointer(msg)))
```

Comment:
This frees the heap memory allocated on the above line at the end of this scope (defered).

Only suggestion here is to make use of the above suggested `hglobal`

```suggestion
	defer win.GlobalFree(hglobal)
```